database/sql/V037__add_order_surplus_fee.sql

```diff
@@ -0,0 +1,3 @@
+CREATE INDEX ON orders (class);
+ALTER TABLE orders ADD COLUMN surplus_fee numeric(78,0) NOT NULL DEFAULT 0;
```

Comment:
Just a meta-question - did you consider using a separate table instead of extending the `orders` table? (not suggesting one or the other, just wondering what your take is).

Reply:
I don't like the idea of introducing a separate table for this. I also don't like e.g. how we have the `onchain_invalidations` table. It makes the code and the queries more complex and I don't see much of a benefit to it.

Reply:
Sounds good.

Reply:
I think the reason why it was done with a separate table is due to a fear of having too many columns on the orders table. But IMO this doesn't matter. The DB is a "detail" (the DB layer only exists to serve the domain logic layer), so if the DB gets a little "ugly" that's not a problem since the data should be mapped to any format convenient for the domain logic anyway.